### ISSUE TITLE
Discuss and review `BATCH` message type proposal (aka `FASTSYNC`)

### ISSUE THREAD

**Issue by aaronsteers** [MeltanoLabs/singer-spec-working-group#2]
As described here: https://gitlab.com/meltano/sdk/-/issues/9

**Comment by judahrand** [comment]
> I think this is a very important feature if someone is ever to migrate from PipelineWise to Meltano

Absolutely, it was a shame that we could not consider Meltano due to the initial size of our tables.

**Comment by hhcomply** [comment]
Hi @aaronsteers this is a great idea! Was this feature ever implemented?

**Comment by aaronsteers** [comment]
@hhcomply re:

> Hi @aaronsteers this is a great idea! Was this feature ever implemented?

Yes! We implement into the Meltano SDK:
- https://sdk.meltano.com/en/latest/batch.html
